.asf.yaml

```diff
@@ -19,7 +19,7 @@ github:
   description: "Apache Kvrocks Website"
   homepage: https://kvrocks.apache.org/
   features:
-    issues: false
+    issues: true
```

Comment:
I'd suggest we use a unified issue track to avoid fragmented discussions. Especially that the website is almost a downstream of the main repo.

Reply:
Yes, I falsely committed this, rollback now.